src/mechanical/fusion.md

```diff
@@ -127,3 +147,76 @@ You'll also be able to do it with a curved ridge (the one made with fillet)
 ![img](img/mechanical/Chamfer_4.png)
 *Note that in this case you'll be limited by the angle of the previous made fillet.*
 
+Another option is to change the "Chamfer type" and select "Distance and angle" to make a chamfer with a selected angle.
+![img](img/mechanical/Chamfer_5.png)
+
+
+### Join
+A joint is a link between two pieces that describes the way they move one on the other. Is comes in handy when describing actions to external people, or when modeling pieces in an assembly to avoid collisions due to design mistakes.
+To create joints, you have to first **ground components**. By grounding components you avoid them to move, and so you can use them as a static reference. Take *Cortex* for example, the base has been grounded and all the rest around him has been jointed.
+![img](img/mechanical/cortex.png)
+This means you can not drag it around. The second thing you have to do is **create rigid groups**. By searching for "rigid group" when hitting the "s" key you will enter the "rigid group" menu. You can now select all the component that should move together. You'll still be able to drag them around, but only as one group and not separately anymore. This is for example the case for the mecanum wheels as you can not move is different pieces separately.
+After this is done, we can now **join modules together** ('j' key). To do so select the points of the modules that will touch after jointing starting with the one you want to be able to move.
+![img](img/mechanical/joint_face1.png)![img](img/mechanical/joint_face2.png)
+
+After this is done you can select the type of joint you want and the axis along which to movement will have to be.
+
+![img](img/mechanical/joint_type.png)![img](img/mechanical/joint_defined.png)
+After confirming the settings, you can right-click on the joint to edit the joints limits and inverse the natural rotation.
+![img](img/mechanical/joint_limit.png)
+
+ To learn more about jointing component click on the link to Lars's video about that topic <a href=" https://www.youtube.com/watch?v=KQNgIfjMr84">  Fusion 360 Tutorial — How to get a handle on Assembly and Joints in Fusion</a>.
+
+
+
+## 3. Importance: dif bodies vs components       PUISS
```

Comment:
Could you remove the annotations, or at least change them into comments that will not be rendered, before we merge into master?